File: src/types/catalog.ts
export interface Sku {
  id: number
  sku: string
  category: string
  gost: string
  d: number | null
  D: number | null
  B: number | null
  type: string
  brand: string
  stock: string
  price: number
  img: string
}

export interface SkuInput {
  sku: string
  category?: string
  gost?: string
  d?: number | null
  D?: number | null
  B?: number | null
  type?: string
  brand?: string
  stock?: string
  price: number
  img?: string
}

File: crmks/backend/schemas/catalog.py
from __future__ import annotations

from typing import Optional

from pydantic import BaseModel


class SkuInput(BaseModel):
    sku: str
    category: Optional[str] = ""
    gost: Optional[str] = ""
    d: Optional[float] = None
    D: Optional[float] = None
    B: Optional[float] = None
    type: Optional[str] = ""
    brand: Optional[str] = ""
    stock: Optional[str] = ""
    price: float = 0
    img: Optional[str] = ""
